### PR TITLE
[civ2] run ray_ci unit tests on civ2 pipeline

### PR DESCRIPTION
**Pull request by aslonnie**
dogfooding our new pipeline with our own tests first.

### CODE REVIEW

**Comment by aslonnie** [comment]
and here you go:

https://buildkite.com/ray-project/oss-ci-blocking/builds/328

and we can make it blocking, so that no one can break it anymore.

**Review by edoakes — APPROVED** [review]
stamp linter change

File: ci/forge/Dockerfile.forge [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+npm install -g @bazel/bazelisk
+ln -s /usr/local/bin/bazel /usr/local/bin/bazelisk
+
+# A non-root user. Use 2000, which is the same as our buildkite agent VM uses.
+adduser --home /home/forge --uid 2000 forge --gid 100
+usermod -a -G docker forge
+
+EOF
+
+USER forge
```
Comment (can-anyscale) [inline ci/forge/Dockerfile.forge:47]: Discuss offline, but it seems that **_ci/ci.sh build_** will fail with this user

**Review by can-anyscale — APPROVED** [review]

File: ci/forge/Dockerfile.forge (hunk 1 above)
Comment (aslonnie) [reply ci/forge/Dockerfile.forge:47]: what is the failure/error.

this is not intended for use to build ray yet, but only for things that are fully converted to use bazel.

building with root will also produce a lot of warnings.

Comment (can-anyscale) [reply ci/forge/Dockerfile.forge:47]: Something about permission defined when creating directory ./python/ray/generated; I'll send you the exact errors if I can recover it

**Pull request merged**